app/build.gradle

```diff
@@ -158,9 +158,13 @@ dependencies {
     //CSV Serialization/Deserialization
     implementation "com.opencsv:opencsv:$rootProject.openCsvVersion"
 
+
     //Database
     implementation "org.greenrobot:greendao:$rootProject.greenDaoVersion"
 
+    //image popup
+    implementation 'com.github.chathuralakmal:AndroidImagePopup:1.2.1'
```

Comment:
There seems to be a lot of extra libraries added into this. It might just be best to use the image viewer that is already build into the app (the same as is used to display the product images). Can you try this rather than using this library?